.test-defs/cmd/shoot-maintenance/main.go

```diff
@@ -129,46 +206,7 @@ func init() {
 func main() {
 	ctx := context.Background()
```

Comment:
I think the whole test should have a timeout so that changes can be cleaned up correctly.
If we only set a timeout on the TestMachinery level, after that timeout the testpod just gets killed with no chance for the defer cleanup function to run.